### ISSUE TITLE
Rec. 22: Develop FAIR components to meet research needs

### ISSUE THREAD

**Issue by sjDCC** [FAIR-Data-EG/Action-Plan#22]
While there is much existing infrastructure to build on, the further development and extension of FAIR components is required. These tools and services should fulfil the needs of data producers and users, and be easy to adopt.

- The development of FAIR compliant components needs to involve scientific communities, technical experts and other stakeholders. They should be provided with a forum for the exchange of views.
**Stakeholders:** Data services; Research communities; Global coordination fora.

- Engagement of the necessary stakeholders and experts needs to be facilitated with appropriate funding, support, incentives and training. 
**Stakeholders:** Funders; Institutions. 
 
- FAIR components will need regular iteration cycles and evaluation processes to ensure that they are fit for purpose and meet community needs.
**Stakeholders:** Data services; Research communities.

**Comment by ChiaraGabella** [comment]
[SIB position](https://www.sib.swiss/): An appreciated alternative would be a simplified short document (“FAIR for dummies”) that describes at in a fast way and at a basic level the requirements to make sure that a resource is as “FAIR” as possible given the financial/personal resource at the disposition of the person/organisation. Such document should be intended to provide support with simple guidelines of fast implementation rather than demanding additional efforts and resources to the scientific community.

**Comment by ghost** [comment]
[4TU.Centre for Research Data ](https://researchdata.4tu.nl/en/home/)position: The Data Steward as stakeholder was mentioned previously in relation to appraisal and selection, working hand in hand with the research community. We are missing them in the last recommendation.

**Comment by katerbow** [comment]
[DFG](http://www.dfg.de) position: Recommendation 22 appears rather unclear with respect to the question, what FAIR components could be. It is welcomed to see that their development should be based on the researcher’s requirements; however, it should be made clear where FAIR components could be of help and support. It is suggested to delete this recommendation.

**Comment by Falco-KUB** [comment]
I agree with katerbow, it should be made clear, what FAIR components are, in particular those that support the whole data life cycle and not only repositories. In general, I think that the aspect of "active" or "dynamic" data is missing in the action plan.

**Comment by bertocco** [comment]
INAF (astronomy) position:
FAIR principles can not be partially applied to a resource. Funding, training, manuals, dissemination are necessary to implement a fair resource. FAIR components can not be domain agnostic, each community should be able to fulfill the three points proposed by this issue.

**Comment by Eefkesmit** [comment]
Contribution on behalf of **the International Association of STM Publishers (STM)**:
We see 4 cornerstone components in a machine-actionable eco system for FAIR Data;  STM and STM publishers offer to collaborate on developing these with other stakeholders, see also our suggestions under recommendation 4: 

1. **Data Availability policies and statements** --  design and implement standardised research data policies for scholarly publications, including Data Availability statements in published articles, preferably with the Research Data Alliance,
2. **Aligning the submission of data and publications** -- promote and enable the use of trusted data repositories for datasets supporting publications, in conjunction with submission of manuscripts where appropriate – via recommended repository lists, services to help deposit data alongside the submission of manuscripts, and technological integrations between scholarly infrastructure, eg by means of API-standards.
3. **Arrange for universal linking between datasets and publications**, bi-directionally-- support adoption and implementation of a SCHOLIX-framework, see www.scholix.org .
4. **Data Citation standards** -- Promote and implement data citation rules and standards according to the recommendations of FORCE11, to provide credit for good data practice.”

STM and STM publishers offer to collaborate with all stakeholders to achieve this.

**Comment by pkdoorn** [comment]
Thumbs up.

**Comment by mromanie** [comment]
[ESO](www.eso.org) position
The involvement of the individual communities and disciplines is crucial here. A global solution may not exist.

**Comment by MSoareses** [comment]
Again in line with @Eefkesmit’s comment above and remitting to my own comment in Rec. https://github.com/FAIR-Data-EG/Action-Plan/issues/4  publishers are also stakeholders as one of the components of the ecosystem.

**Comment by gtoneill** [comment]
Some overlap with Recommendation 4 related to FAIR components. Perhaps merge?
